ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v2/title/title.html

```diff
@@ -0,0 +1,24 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2024 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+  <div data-sly-use.title="com.adobe.cq.forms.core.components.models.form.FormTitle"
+  data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+  data-sly-test.text="${title.text}"
+  id="${title.id}"
+  data-cmp-data-layer="${title.data.json}"
+  class="cmp-title">
```

Comment:
BEM markup should align with other components of adaptiveform, please change the same.

Reply:
Done!